Add card to contact vaadin team for developing custom components / sp… (#4063)
vaadin/docs: articles/components/index.adoc

[.sr-only]
<<{articles}/flow/create-ui/web-components#,Integrate Web Components>>

[.card]

> === Contact Vaadin Taem

Reviewer: ```suggestion
=== Contact Vaadin Team
```

image::{articles}/flow/_images/web-components.svg[opts=inline, role=icon,32,32]

> We would love to hear more about what kind of a component you are looking for. [Please share your thoughts](https://pages.vaadin.com/contact).
> 
> [.sr-only]
> <<{articles}/flow/create-ui/web-components#,Contact Vaadin Team>>

Reviewer: Shouldn't have a link within the content links, since the card itself acts as a link already.

```suggestion
We'd love to hear what kind of components you are looking for. Share your ideas!

[.sr-only]
link: https://pages.vaadin.com/contact,Contact Vaadin Team>>
```

Is this the right place to guide users? Would a GitHub issue be more expected, than a formal contact form? @rolfsmeds, what do you think – would you like another CTA for users to share their component ideas? Or is this more for sales (marketing?) to get leads?

Reviewer 2: If these suggestions go via Sales, then Sales needs to pass it on to us, and then we don't have a convenient 2-way channel to ask more questions (which we nearly always want to do).

Reviewer: This starts to smell stale. Anyone wanna continue working on this?
